radixdlt-java/src/main/java/com/radixdlt/client/atommodel/tokens/TokenParticle.java

```diff
@@ -12,17 +13,23 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
+import com.radixdlt.client.core.crypto.ECKeyPair;
+import com.radixdlt.client.core.crypto.ECPublicKey;
 import org.radix.serialization2.DsonOutput;
 import org.radix.serialization2.DsonOutput.Output;
 import org.radix.serialization2.SerializerId2;
 import org.radix.utils.UInt256;
 
 @SerializerId2("TOKENCLASSPARTICLE")
-public class TokenParticle extends Particle implements Identifiable {
+public class TokenParticle extends Particle implements Identifiable, Ownable {
 	@JsonProperty("address")
 	@DsonOutput(Output.ALL)
 	private RadixAddress address;
 
+	@JsonProperty("owner")
+	@DsonOutput(Output.ALL)
+	private ECPublicKey owner;
```

Comment:
The corresponding `owner` field in Core is an `ECKeyPair` as opposed to `ECPublicKey`. Is this divergence by intention?

Reply:
The divergence is intentional, not wanted but necessary. Core currently doesn't have a corresponding ECPublicKey so it is mapped like it is now.